README.md

```diff
@@ -17,20 +18,22 @@ for information about the standalone version of cri-containerd.
 With it, you could run Kubernetes using containerd as the container runtime.
 ![cri](./docs/cri.png)
 ## Current Status
+`cri` is a native built-in plugin of containerd >= 1.1.
+
 `cri` is in beta:
 * It is feature complete.
 * It (the beta version) works with Kubernetes >= 1.9.
 * It has passed all [CRI validation tests](https://github.com/kubernetes/community/blob/master/contributors/devel/cri-validation.md).
-* It has passed all regular [node e2e tests](https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-node-tests.md).
-* It has passed all regular [e2e tests](https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md).
+* It has passed all [node e2e tests](https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-node-tests.md).
+* It has passed all [e2e tests](https://github.com/kubernetes/community/blob/master/contributors/devel/e2e-tests.md).
 
 See [test dashboard](https://k8s-testgrid.appspot.com/sig-node-containerd)
 ## Support Metrics
 | CRI-Containerd Version | CRI Plugin Version | Kubernetes Version |
 |:----------------------:|:------------------:|:------------------:|
 |     v1.0.0-alpha.x     |                    |      1.7, 1.8      |
 |      v1.0.0-beta.x     |                    |        1.9         |
-|                        |        HEAD        |        1.10+       |
+|        DEPRECATED      |        HEAD        |        1.10+       |
```

Comment:
see above.. can we use another term other than deprecated? or just leave blank?

Reply:
Done.